test/extensione2e/kafka/user_permissions_test.go

```diff
@@ -49,31 +52,62 @@ func TestKafkaUserPermissions(t *testing.T) {
 		},
 	}
 
+	kafkaChannel := &kafkachannelv1beta1.KafkaChannel{
+		Spec: kafkachannelv1beta1.KafkaChannelSpec{
+			NumPartitions:     1,
+			ReplicationFactor: 1,
+		},
+	}
+
+	kafkaSink := &kafkasinksv1alpha1.KafkaSink{
+		Spec: kafkasinksv1alpha1.KafkaSinkSpec{
+			Topic:             "my-topic",
+			NumPartitions:     pointer.Int32Ptr(10),
```

Comment:
How about using `ptr.Int32(10)` and import "knative.dev/pkg/ptr" instead?
Same for ` ptr.String(kafkasinksv1alpha1.ModeStructured)`.

Reply:
I could do that. The two functions have absolutely same result. Just for my understanding - is there any benefit?

Reply:
Avoid introducing new deps.